styles/widgets/generic/color-schemes/carmine/generic.carmine.less

```diff
@@ -1566,4 +1566,4 @@
 @filemanager-breadcrumbs-item-focus-color: @base-text-color;
 @filemanager-custom-thumbnail-color: @base-border-color;
 @filemanager-file-item-select-color: @base-bg;
-@filemanager-file-item-select-bg: @base-accent;
+@filemanager-file-item-select-bg: @treelist-selection-bg;
```

Comment:
Do not use constants from other widgets, because their values can be changed by the customer in the process of creating new themes using a special tool.

Reply:
Does in mean that I can only inherit @base* constants?

Reply:
Yes, base constants are common for all widgets. But you can also create your own constants, specific for FileManager widget.